db/migrate/20240526172112_create_census_data.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCensusData < ActiveRecord::Migration[7.0]
+  def change
+    create_table :census_data do |t|
+      t.references :statistic, null: false, foreign_key: { to_table: :census_statistics }
+      t.string :amount
```

Comment:
I'm not sure about being a string
It looks like only 10 variables come is as strings from the census and it's "Median year". I'm not sure why a year would be a string and not an int...
it's easy enough to cast for now though and then we can do an analysis for any weird values that might warrant us keeping it a string and convert the column back later.